exercises/practice/pangram/.approaches/introduction.md

```diff
@@ -16,18 +15,23 @@ ispangram(input) = 'a':'z' ⊆ lowercase(input)
 
 ⊆ can be entered at the Julia repl as `\subseteq` followed by tab. It's the infix operator for the `issubset` function.
 
-## Two more one-liners
+## Approach: using currying
 
-The first uses function currying, which may look strange at first, but is very convenient for predicates like `in`.
-The second uses an anonymous function and avoids taking a copy of the input to reduce memory usage.
+This approach uses function currying, which may look strange at first, but is very convenient for predicates like `in`.
 
 ```julia
 ispangram1(input) = all(in(lowercase(input)), 'a':'z')
+```
+
+## Approach: use an anonymous function
 
+The second uses an anonymous function and avoids taking a copy of the input to reduce memory usage.
+
+```julia
 ispangram2(input) = all(c -> in(c, input) || in(uppercase(c), input), 'a':'z')
 ```
 
-## Going faster with bytes
+## Approach: bitwise tracking
```

Comment:
```suggestion
## Approach: marking bytes
```

Kinda bikesheddy, but I like that this and "counting bytes" have similar names because they're related approaches.